### DISCUSSION TITLE
Mezzanine 6.0.0 got an error  for the EXTRA_MODEL_FIELDS

### DISCUSSION THREAD

**Discussion by bongkot** [stephenmcd/mezzanine#2036] [General]
Mezzanine 6.0.0 got an error when I add "mezzanine.core.fields.FileField" to EXTRA_MODEL_FIELDS eg.
EXTRA_MODEL_FIELDS = (
   (
        "mezzanine.pages.models.RichTextPage.header_image",
         "mezzanine.core.fields.FileField",
        ("Header Image",),
        {"blank": True, "upload_to": "pages"},
    ),
)

**Comment by kenbolton** [comment]
Traceback?

**Reply by bongkot** [reply]
When add that EXTRA_MODEL_FIELDS for "mezzanine.core.fields.FileField" I got problem below. But when I change that field to "django.db.models.FileField", the issue has resolve.
```

File "D:\Workspace\tooManyV3Env\lib\site-packages\mezzanine\utils\sites.py", line 6, in <module>
    from django.contrib.sites.models import Site
  File "D:\Workspace\tooManyV3Env\lib\site-packages\django\contrib\sites\models.py", line 79, in <module>
    class Site(models.Model):
  File "D:\Workspace\tooManyV3Env\lib\site-packages\django\db\models\base.py", line 132, in __new__
    raise RuntimeError(
RuntimeError: Model class django.contrib.sites.models.Site doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

**Reply by MelvinDRM** [reply]
Hello:

Any fix or workaround for this issue?
